.dockerignore

```diff
@@ -0,0 +1,7 @@
+.devcontainer
+.github
+.mypy_cache
+deploy-scripts
+examples
```

Comment:
WHy are all the others in the .dockerignore?

Reply:
Because I copied it from another project and forgot to clean it up nicely.